src/Settings/SettingsPage.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { Settings } from '@folio/stripes/smart-components';
+import CategorySettings from './CategorySettings';
+
+class SettingsPage extends Component {
```

Comment:
this component doesn't contain any state, it could be easily written as SFC to reduce code lines amount